app/ic.h

```diff
@@ -23,13 +26,26 @@ class IC : public GraphicElement
     IC(QGraphicsItem *parent = nullptr);
     ~IC() override;
 
+    //!
+    //! \brief save saves the IC's internal file to the given data stream
+    //!
     void save(QDataStream &ds) const override;
     void load(QDataStream &ds, QMap<quint64, QNEPort *> &portMap, double version) override;
+    //!
+    //! \brief loadFile is used when an IC is dragged into the editor
+    //!
     void loadFile(const QString &fname);
+    //! \brief getFile returns this->m_file
```

Comment:
this is a kind of documentation that's not really userfull.
perhaps: "returns the current file name that represents this IC" ?